tests/src/JIT/CodeGenBringUpTests/LocallocCnstB1.cs

```diff
@@ -0,0 +1,48 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+
+
+using System;
+using System.Runtime.CompilerServices;
+public class BringUpTest
+{
+    const int Pass = 100;
+    const int Fail = -1;
+
+    // Reduce all values to byte
+    public static unsafe bool CHECK(byte check, byte expected) {
+        return check == expected;
+    }
+
+    [MethodImplAttribute(MethodImplOptions.NoInlining)]
+    public static unsafe int LocallocCnstB1()
+    {
+        byte* a = stackalloc byte[1];
+        for (int i = 0; i < 1; i++)
+        {
+            a[i] = (byte) i;
+        }
+
+        for (int i = 0; i < 1; i++)
+        {
+            if (!CHECK(a[i], (byte) i)) return i;
+        }
+
+        return -1;
+    }
+
```

Comment:
It generates for `LocallocCnstB1.cs` test with `\o+` following code:
```ASM
000026  2300           movs    r3, 0
000028  B408           push    {r3}
00002A  B408           push    {r3}
00002C  466B           mov     r3, sp
00002E  F8C9 3004      str     r3, [r9+4]
```
Allocated array (with size 8, which is stack alignment) is located in `[r9+4]`, legacy backend does the same.
But `CHECK()` method was inlined, I've added `NoInline` attribute to fix it.